policy/framework.go

```diff
@@ -25,19 +25,37 @@ type ResolvedFrameworkNode struct {
 	Type ResolvedFrameworkNodeType
 }
 
+type ResolvedFrameworkReferenceSet struct {
+	items map[string]struct{}
```

Comment:
if it only has this set of items, why not just `type ResolvedFrameworkReferences map[string]struct`? saves the struct and the pointer

Reply:
changed